instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -63,14 +63,14 @@
 
 
 def trace_integration(
-    connect_module: typing.Callable[..., typing.Any],
-    connect_method_name: str,
-    database_system: str,
-    connection_attributes: typing.Dict = None,
-    tracer_provider: typing.Optional[TracerProvider] = None,
-    capture_parameters: bool = False,
-    enable_commenter: bool = False,
-    db_api_integration_factory=None,
+        connect_module: typing.Callable[..., typing.Any],
```

Comment:
Please run `tox -e ruff` or setup pre-commit to avoid whitespaces changes https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md#development